dump/src/test/scala/org/dbpedia/extraction/dump/ExtractionTestAbstract.scala

```diff
@@ -0,0 +1,162 @@
+    mapper.registerModule(DefaultScalaModule)
+    val json = mapper.writeValueAsString(content)
+
+     val targetDir = new File("../dump/test_logs/")
+    // create directories
+    targetDir.mkdirs()
+      
+    if( Files.exists(Paths.get(urisListUsed))){
+
+      val Urifile = Source.fromFile(urisListUsed)
+      val fileContents = Urifile.getLines.mkString
+      Urifile.close()
+
+      fileName2="../dump/test_logs/"+fileName+"_"+fileContents+"_"+today+".log"
+
+    }else{
+      fileName2="../dump/test_logs/"+fileName+"_"+"base-list"+today+".log"
+
+    }
+    val file = new File(fileName2)
+    val bw = new BufferedWriter(new FileWriter(file))
+    bw.write(json)
+    bw.close()
+  }
+
+  def extract(config: Config, jobsRunning: ConcurrentLinkedQueue[Future[Unit]]): Array[Map[String,String]]  ={
+    println(">>>>>>>>> EXTRACT  - BEGIN")
+    var  mapResults = Array[Map[String,String]]()
+    val configLoader = new ConfigLoader(config)
+
+    val parallelProcesses = 1
+    println(parallelProcesses)
+    val jobs=configLoader.getExtractionJobs
+    println(">>>>>>>>> EXTRACT - NBJOBS > "+jobs.size)
+    println("LAUNCH JOBS")
+    for (job <- jobs) {
+
+      job.run()
+
+      val lang=job.extractionRecorder.language
+      val records=job.extractionRecorder
+      println(">>>>>>>>> EXTRACT - LANG > " + lang.wikiCode)
+
+      val status = records.getStatusValues(lang);
+      var numberOfFailedPages429 = 0
+      var numberOfFailedPages503 = 0
+      var numberOfFailedPagesIOException = 0
+      var numberOfFailedPagesOutOfMemoryError = 0
+      var numberOfFailedPagesNullPointerException = 0
+      var mapLocal= status
+
+      mapLocal += "language" -> lang.wikiCode.toString();
+
+      try {
+        val listFailedPages_ = records.listFailedPages(lang)
+        for( failed <- listFailedPages_) {
+          if(failed.toString().contains("Server returned HTTP response code: 429")){
+            numberOfFailedPages429 += 1
+          }
+          if (failed.toString().contains("Server returned HTTP response code: 503")){
+            numberOfFailedPages503 += 1
+          }
+          if (failed.toString().contains("java.io.IOException")){
+            numberOfFailedPagesIOException += 1
+          }
+          if (failed.toString().contains("java.io.OutOfMemoryError")){
+            numberOfFailedPagesOutOfMemoryError += 1
+          }
+          if (failed.toString().contains("java.io.NullPointerException")){
+            numberOfFailedPagesNullPointerException += 1
+          }
+        }
+      } catch {
+        case e: Exception =>  None
+      }
+
+      mapLocal += "numberOfFailedPages429" -> numberOfFailedPages429.toString
+      mapLocal += "numberOfFailedPages503" -> numberOfFailedPages503.toString
+      mapLocal += "numberOfFailedPagesIOException" -> numberOfFailedPagesIOException.toString
+      mapLocal += "numberOfFailedPagesOutOfMemoryError" ->numberOfFailedPagesOutOfMemoryError.toString
```

Comment:
for code style consistency with neighboring lines
```suggestion
      mapLocal += "numberOfFailedPagesOutOfMemoryError" -> numberOfFailedPagesOutOfMemoryError.toString
```